.eleventy.js

```diff
@@ -50,6 +50,46 @@ module.exports = function (eleventyConfig) {
   eleventyConfig.addCollection('sortedEntries', collectionApi =>
     getSortedCollection(collectionApi, 'entries'),
   );
+  eleventyConfig.addCollection('indexItems', collectionApi => {
```

Comment:
We can clean this up into an individual file once we decide on slugs or title case